pkg/networkmanager/dialogs-common.jsx

```diff
@@ -198,10 +201,27 @@ export const NetworkAction = ({ buttonText, iface, connectionSettings, type }) =
         if (type == 'vlan') settings = getVlanGhostSettings();
         if (type == 'team') settings = getTeamGhostSettings({ newIfaceName });
         if (type == 'bridge') settings = getBridgeGhostSettings({ newIfaceName });
+        if (type == 'wireguard') settings = getWireGuardGhostSettings({ newIfaceName });
     }
 
     const properties = { connection: con, dev, settings };
 
+    async function resolveDepsAndShowDialog(dlg) {
+        try {
+            await cockpit.script("command -v wg");
+            Dialogs.show(dlg);
+        } catch {
```

Comment:
Please don't include the `Dialogs.show()` inside the try/catch. That ignores *every* error, so must be very confined. It's also called again in the following code, so please DRY and just move it down.